coralproject/talk
# [CORL-2611]: update external media width in comments

## What does this PR do?

These changes update the width of external media images within comments. The widths now match what we are doing for Youtube (75% width on desktop, 100% width on mobile). 

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Add images to your comments. See that they are shown at 75% width on desktop and 100% width on mobile in the stream. Also take a look at comments with images in the admin; their widths have been updated to just be 100%.
 
## How do we deploy this PR?
